sql/cache.sql

```diff
@@ -14,3 +14,7 @@ CREATE TABLE IF NOT EXISTS  _timescaledb_cache.cache_inval_extension();
 -- not actually strictly needed but good for sanity as all tables should be dumped.
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_cache.cache_inval_hypertable', '');
 SELECT pg_catalog.pg_extension_config_dump('_timescaledb_cache.cache_inval_extension', '');
+
+GRANT SELECT ON TABLE _timescaledb_cache.cache_inval_extension TO PUBLIC;
```

Comment:
As the comment suggests, it is actually not clear why we are dumping these tables to begin with. They are only there for cache invalidation purposes and don't actually have any data (and never should have). 

Seems to me that a better solution is actually to not include these in the dump since the they should already be created by create extension.

Reply:
Btw, it's weird and inconsistent that we need GRANTs  on both the `_timescaledb_cache` schema in `tables.sql` and the explicit tables here. 

If we still want to dump these tables, I think this file should either be moved to `pre_install` or we should create these table in `tables.sql` so that we handle all tables and permissions in one place. I think probably the latter is the most consistent and least error prone.

But, IMHO, like my previous comment suggests, these are internal objects that should not be part of the dump.

Reply:
Even though i didnt register them for dumping anymore, pg_dump is still trying to dump them. So the permissions are indeed required for pg_dump to work.